actions/admin/destinations.go

```diff
@@ -10,8 +10,17 @@ import (
 )
 
 // destinationsSearch will fetch a list of destinations filtered by metadata
+// Search for destinations filtering by metadata godoc
+// @Summary      Search for destinations
+// @Description  Search for destinations
+// @Tags		 Admin
+// @Produce      json
+// @Param metadata query string false "Metadata filter"
+// @Param conditions query string false "Conditions filter"
+// @Success      200
+// @Router       /v1/admin/destinations/search [post]
+// @Security bux-auth-xpub
 func (a *Action) destinationsSearch(w http.ResponseWriter, req *http.Request, _ httprouter.Params) {
-
 	// Parse the params
 	params := apirouter.GetParams(req)
 	queryParams, metadata, conditions, err := actions.GetQueryParameters(params)
```

Comment:
There is also such thing as queryParams, which is used in "search" endpoints. So need to add 4 of it's params to @Param

Reply:
You can specify ordering and pages. You can check, how it was added for search of utxo